app/src/main/java/com/zeapo/pwdstore/pwgen/PasswordGenerator.kt

```diff
@@ -0,0 +1,125 @@
+package com.zeapo.pwdstore.pwgen
+
+import android.content.Context
+import android.content.SharedPreferences
+
+import java.util.ArrayList
+
+object PasswordGenerator {
+    internal val DIGITS = 0x0001
+    internal val UPPERS = 0x0002
+    internal val SYMBOLS = 0x0004
+    internal val AMBIGUOUS = 0x0008
+    internal val NO_VOWELS = 0x0010
+
+    internal val DIGITS_STR = "0123456789"
+    internal val UPPERS_STR = "ABCDEFGHIJKLMNOPQRSTUVWXYZ"
+    internal val LOWERS_STR = "abcdefghijklmnopqrstuvwxyz"
+    internal val SYMBOLS_STR = "!\"#$%&'()*+,-./:;<=>?@[\\]^_`{|}~"
+    internal val AMBIGUOUS_STR = "B8G6I1l0OQDS5Z2"
+    internal val VOWELS_STR = "01aeiouyAEIOUY"
+
+    // No a, c, n, h, H, C, 1, N
+    private val pwOptions = "0ABsvy"
+
+    /**
+     * Sets password generation preferences.
+     *
+     * @param ctx     context from which to retrieve SharedPreferences from
+     * preferences file 'PasswordGenerator'
+     * @param argv    options for password generation
+     * <table summary="options for password generation">
+     * <tr><td>Option</td><td>Description</td></tr>
+     * <tr><td>0</td><td>don't include numbers</td></tr>
```

Comment:
it would be interesting to only include numbers. Adding an option "don't include letters".

Reply:
Sounds good! I'll pitch the feature in a follow-up PR once this has landed.